packages/lesmis-server/src/routes/TranslatableRoute.ts

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
Does anything about this file make it specific to LESMIS? I'd be keen to stick it in `server-boilerplate`, I'm sure we'll need it there at some point so might as well avoid the risk of coupling it too tightly to lesmis-server in the meantime